chapters/1.md

```diff
@@ -11,31 +11,31 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
 
-Depending on how the funds were sent, it could take anywhere from minutes to days before the $2,500 shows up in George's bank account. Since George is not privy to the banks' ledgers or communications, the entire process is opaque and cannot be monitored.
+Depending on how the funds were sent, it could take anywhere from minutes to days before the $2,500 shows up in George's bank account. Since George is not privy to the banks' ledgers or internal communications, the entire process is opaque to him and cannot be monitored.
 
 Many people that have not personally experienced economic disruption take functioning banks and the validity of their IOUs for granted. Few individuals consider the unsettling ramifications of handing their lifelong savings to opaque corporations, often putting all their eggs in a single institutional basket. 
 
-Losses can occur due to many other issues. One clear example could be the neglicence of the bank operator (while, for example, shifting assets between two accounts). A bank operator had to face up work overloading, lacking of standards, and poor working conditions; even if they developed a system to automatically check for errors, some recent news reports prove the contrary. We have to mark that banks are based on partial inexistent fund and if all people widthdraw their money, the bank overextends their assets and goes out of business. Feelings of malice and corruption have also an impact on economical systems; a hacker thieves or a rogue employe could profit off their priveleges in order to drain all your assets.
+Losses can occur due to many other issues. Bank employees may simply make a mistake and monitoring software may fail to catch errors. Bank employees, IT staff and third party partner companies might take advantage of their privileged position for personal gain, thus creating a much larger group which must be trusted when using a bank. Additionally, banks usually have debt obligations which far exceed the cash they hold and would collapse if even a small portion of their customers decided to withdraw their money at the same time (a so-called “bank run”).
 
-The availability of the bank have been always an issue for the customer: especially in the weekends and during national holidays (which varies from nation to nation), there are some practical difficulties due to closing of them. Many economical transactions sent in "bank holidays" could fail and you have to wait the opening to receive or send your funds. 
+Banks limit your access to your money. They may be closed on weekends and when there is large emergencies or during holidays (which vary from country to country). If the bank's servers are shut down for maintenance or there is a security issue in the e-banking system or application, customers have no recourse but to wait until it is made available again. This can lead to significant delays, frustration and loss of economic opportunities by those who rely on the banks.
```

Comment:
> and when there is large emergencies

I suppose you wanted to write: there are

Reply:
Done.